### ISSUE TITLE
Have you checked out csv-geo-au for loading common boundaries?

### ISSUE THREAD

**Issue by tobybellwood** [Denubis/gisDigitalRights#1]
Had a quick look at your NationalMap Feedback issue, It'll almost definitely be a size timeout given the ~60MB file loading client side.

Given you seem to be loading regular electoral boundaries loaded with information, you can make use of the csv-geo-au feature on NationalMap (documentation still a work in progress, but at https://github.com/TerriaJS/nationalmap/wiki/csv-geo-au)

Basically, as long as you call the boundaries in your csv with a set title (in this case com_elb_name_2016), you can load info into NationalMap with tiny loads.

I've attached a sample file (the 2016 2pp by div) - just remove the .txt from the name on download, and you'll be able to drag and drop it onto NationalMap (caveat being the issue I've just logged at TerriaJS/terriajs#2330 where Flynn doesn't appear for some reason)

Basically all ABS regions, LGAs, POA and electoral boundaries *should* work - we're continually adding more.  The wiki (above) covers most of them (but the full list is through https://github.com/TerriaJS/nationalmap/blob/master/wwwroot/data/regionMapping.json

Let us know if you need any more help, or think we can add something else - data@pmc.gov.au, or throw an issue into https://github.com/TerriaJS/nationalmap/issues

[HouseTppByDivisionDownload-20499.csv.txt](https://github.com/Denubis/gisDigitalRights/files/706448/HouseTppByDivisionDownload-20499.csv.txt)

**Comment by Denubis** [comment]
Cheers! I sent an e-mail with fundamental goals of what I'm doing. Thanks so much for responding so quickly.
